tcms/core/management/commands/initial_setup.py

```diff
@@ -20,4 +24,21 @@ def handle(self, *args, **kwargs):
         self.stdout.write("\n4. Setting permissions:")
         call_command("refresh_permissions", "--verbosity=%i" % kwargs["verbosity"])
 
+        self.stdout.write(" %i " % admins[0].id)
+
+        if 'tcms_tenants' in settings.INSTALLED_APPS:
+            self.stdout.write("\n5. Creating the public tenant:")
+            superusers = User.objects.filter(is_superuser=True).order_by('id')
+            call_command("create_tenant",
+                         "--verbosity=%i" % kwargs["verbosity"],
+                         "-schema_name public",
+                         "--name 'Public tenant'",
+                         "--paid_until 2050-12-31",
+                         "--publicly_readable False",
+                         "--owner_id %i" % superusers[0].id,
```

Comment:
Use `.pk`, not `.id`.  `.pk` mean use the primary key, even if it is not the ID column.